scripts/ps.py

```diff
@@ -17,17 +17,43 @@
 
 import psutil
 
+STATUS_RUNNING = "running"
+STATUS_SLEEPING = "sleeping"
+STATUS_DISK_SLEEP = "disk-sleep"
+STATUS_STOPPED = "stopped"
+STATUS_TRACING_STOP = "tracing-stop"
+STATUS_ZOMBIE = "zombie"
+STATUS_DEAD = "dead"
+STATUS_WAKE_KILL = "wake-kill"
+STATUS_WAKING = "waking"
+STATUS_IDLE = "idle"  # FreeBSD, OSX
+STATUS_LOCKED = "locked"  # FreeBSD
+STATUS_WAITING = "waiting"  # FreeBSD
+STATUS_SUSPENDED = "suspended"  # NetBSD
+
+PROC_STATUSES_RAW = {
+    STATUS_RUNNING:             "R",
+    STATUS_SLEEPING:            "S",
+    STATUS_DISK_SLEEP:          "D",
+    STATUS_STOPPED:             "T",
+    STATUS_TRACING_STOP:        "t",
+    STATUS_ZOMBIE:              "Z",
+    STATUS_DEAD:                "X",
+    STATUS_WAKE_KILL:           "K",
+    STATUS_WAKING:              "W"
```

Comment:
these are not PEP-8 compliant. Redefine the above by removing unnecessary spaces (`STATUS_WAKING: "W"` etc.)